dart/test.dart

```
@@ -0,0 +1,57 @@
+#!/usr/bin/env dart
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'lib/std_uritemplate.dart';
+
+Future<void> main(List<String> args) async {
+  if (args.length != 2) {
+    stderr.write("Usage: test.dart <template_file> <data_file>\n");
+    exit(1);
+  }
+
+  final templateFile = args[0];
+  final dataFile = args[1];
+
+  final dynamic data;
+  try {
+    data = jsonDecode(File(dataFile).readAsStringSync());
+  } catch (e, stack) {
+    stderr.write("Error loading data file: $e\n$stack\n");
+
+    print("false");
```

missed this as well, in this case, we should throw the exception and exit without printing a valid output.

---

Fixed